### PR TITLE
add simple guide to build project in readme.md

### PR DESCRIPTION
**Pull request by raphamorim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,16 @@ Dexter is supported and tested on:
 
 The project documentation files are located in https://github.com/leobalter/DexterJS/tree/master/docs
 
+## Build
+
+Install Grunt and external dependencies. First, install the grunt-cli if you haven't before. This should be installed globally (like this: `npm install -g grunt-cli bower`). Then:
```
Comment (leobalter) [inline README.md:45]: let's omit bower here.

File: README.md [hunk 2]
```diff
@@ -40,6 +40,16 @@ Dexter is supported and tested on:
...
+
+	npm install
+
+Build distribution files.
+
+	grunt -v
```
Comment (leobalter) [inline README.md:51]: this can be only `grunt` to simplify

**Comment by leobalter** [comment]
Would be better if we include two things:
- A reference link on the getting started section (a new bullet item mentioning to build your own copy).
- A intro part mentioning to clone this repo.

**Comment by raphamorim** [comment]
Gotcha @leobalter, i've got a question: Did you consider build and run tests using Docker? 

If yes, can I send a PR w/ this improvement?

**Comment by leobalter** [comment]
I've done the suggestions on Readme. :)

What is docker? (searching on duckduckgo...)

**Comment by raphamorim** [comment]
With a docker file, you can run the project over a image from defined system. This eliminates problems with workspace and other things.

Check out: https://www.docker.com/whatisdocker/

**Comment by leobalter** [comment]
It doesn't look necessary at this point. Sometimes I just avoid things that could increase complexity without much value return. If you insist, I can try it, but at this moment my targets are the xhr2 and timer issues.

**Comment by raphamorim** [comment]
I understand. So i'll focus in xhr2 and timer issues :octocat:

**Pull request merged**